client/my-sites/email/email-providers-stacked-comparison/style.scss

```
@@ -14,8 +9,12 @@
 	}
 }
 
-.email-providers-stacked-comparison__how-they-compare {
-	display: none;
+.email-providers-stacked-comparison__header {
+	font-size: 2rem;
```

Nit: I think we should be using a standard font variable for this value as per [the typography docs](https://wpcalypso.wordpress.com/devdocs/typography).
```suggestion
	font-size: $font-size-large;
```

---

I'll update that.